packages/nextjs/components/Header.tsx

```diff
@@ -25,6 +25,11 @@ export const menuLinks: HeaderMenuLink[] = [
     href: "/debug",
     icon: <BugAntIcon className="h-4 w-4" />,
   },
+  {
+    label: "Profile",
+    href: "/builders/0x9dF97939e2aA22954CFddcD6a57008DA5d5F38Be/",
+    icon: <UserIcon className="h-4 w-4" />,
+  },
```

Comment:
You can remove this link. There's another task that will handle linking all profiles in a more structured way.